samples/bot-types-of-cards/nodejs/bots/botsuggestedCards.js

```diff
@@ -0,0 +1,213 @@
+
+                    case "ThumbnailCard":
+                        await context.sendActivity({ attachments: [ThumbnailCard] });
+                        break;
+
+                    case "SignIn":
+                        await context.sendActivity({ attachments: [this.sendOAuthCard()] });
+                        break;
+
+                    case "OAuth":
+                        await context.sendActivity({ attachments: [this.sendOAuthCard()] });
+                        break;
+
+                    case "ListCard":
+                        await context.sendActivity({ attachments: [ListCard] });
+                        break;
+
+                    case "CollectionCard":
+                        await context.sendActivity({ attachments: [this.sendCollectionCard()] });
+                        break;
+                }
+
+                await context.sendActivity(`You have Selected <b>${text}</b>`);
+            }
+
+            // After the bot has responded send the suggested Cards.
+            await this.sendSuggestedCards(context);
+
+            // By calling next() you ensure that the next BotHandler is run.
+            await next();
+        });
+    }
+
+    /**
+     * Send a welcome message along with suggested actions for the user to click.
+     * @param {TurnContext} turnContext A TurnContext instance containing all the data needed for processing this conversation turn.
+     */
+    async sendWelcomeMessage(turnContext) {
+        const { activity } = turnContext;
+
+        // Iterate over all new members added to the conversation.
+        for (const idx in activity.membersAdded) {
+            if (activity.membersAdded[idx].id !== activity.recipient.id) {
+                const welcomeMessage = `Welcome to Cards. This bot will introduce you to suggested cards. Please select the cards from given options` +
+                    'Please select an option:';
+
+                await turnContext.sendActivity(welcomeMessage);
+
+                //send the suggested Cards.
+                await this.sendSuggestedCards(turnContext);
+            }
+        }
+    }
+
+    // Adaptive Card 
+    sendAdaptiveCard() {
+        return CardFactory.adaptiveCard(AdaptiveCard);
+    }
+
+    // Office356 Connector Card 
+    sendOffice356Card() {
+        return CardFactory.o365ConnectorCard(Office365ConnectorCard);
+    }
+
+    //OAuthCard
+    sendOAuthCard() {
+        return CardFactory.oauthCard(
+            'Sign In',
+            'OAuth connection',// Replace with the name of your Azure AD connection
+            'BotFramework OAuth Card'
+        );
+    }
+
+    //SignInCard
+    sendSignInCard() {
+        return CardFactory.signinCard(
+            'Sign In',
+            'Azure AD connection', // Replace with the name of your Azure AD connection
+            'BotFramework SignIn Card'
+        );
+    }
+
+    //ReceiptCard
+    sendReceiptCard() {
+        return CardFactory.receiptCard(ReceiptCard);
+    }
```

Comment:
Is this workng?

Reply:
this card is not working, will remove it.
Fixed